### ISSUE TITLE
Please commit your code

### ISSUE THREAD

**Issue by sonylnagale** [ellamih/HowsMyKid#1]
Hi Ella, 

Would you please do a `git push origin howsmykid` to commit your code to this repository so I can take a look?

Thanks!

**Comment by ellamih** [comment]
That didn't work. I did instead: 
git add .
git commit .
git push .

**Comment by ellamih** [comment]
I don't understand the diff'ce  between this and my Heroku, for example, when I do: 
git commit -a -m "message" 
git push heroku master

Does Git get updated too?

**Comment by sonylnagale** [comment]
Right. So you have to `git push origin master` as well. There is a difference in that heroku in this instance is an _endpoint_ to push to, just like _origin_ is. And _origin_ in this case is Github.

**Comment by ellamih** [comment]
Did that (git push origin master)

Then did git push origin howsmykid and got:
error: src refspec howsmykid does not match any.
error: failed to push some refs to 'https://github.com/ellamih/HowsmyKid.git'

**Comment by sonylnagale** [comment]
Try `git push -u origin howsmykid`

**Comment by sonylnagale** [comment]
-u is for the first time you push a branch.

**Comment by ellamih** [comment]
I have a quick question. My hows-my-kid-site.heroku.com defaults to /facebooklogin.html however, my latest file is called "/tellthestory.html" (Yup I stupidly changed it) --> how do I fix this? I tried Googling this and didn't see a reasonably understandable explanation.

**Comment by sonylnagale** [comment]
Once you push your code I'll be able to help you fix that.

**Comment by ellamih** [comment]
git push -u origin howsmykid didn't work
Same error. I have Heroku connected (and Git connected on Heroku) so maybe I just have to use the Heroku commands?

**Comment by ellamih** [comment]
I'm going to go to Github desktop and try a push there :)

**Comment by sonylnagale** [comment]
If that doesn't work ping me at 212.887.5853 and we'll do a screenshare.

**Comment by ellamih** [comment]
I did it, but do you see a change?

I wish there was a step-by-step explanation for whats going on between these origins, branches, masters, etc etc! I'm confused by all the branches.

**Comment by ellamih** [comment]
I can join room now! I'll join and see if you can too.

**Comment by ellamih** [comment]
**\* join ZOOM now

**Comment by ellamih** [comment]
Added code to update Firebase, but Firebase is still showing: 
https://hows-my-kid.firebaseio.com/ 
{
  "rules": {
    ".read": "auth != null",
    ".write": "auth != null"
  }
}

**Comment by ellamih** [comment]
Sonyl,
The other area where I need help and might have done too much is in the Stories.html and Stories.js. I am using Handlebars but am trying to not only store comments and then load them, but do so using Handlebars.

Any help or guidance would be appreciated!

**Comment by ellamih** [comment]
I updated my code for storing the TellTheStory.html form in Firebase and still see this in my Firebase Console:

{
  "rules": {
    ".read": "auth != null",
    ".write": "auth != null"
  }
}

**Comment by sonylnagale** [comment]
So in Database->Rules you can actually edit those rules. So I turned
everything to public read/write.

---

Sonyl Nagale sonyl@nagale.com
339 W. 20th St., #2F
New York / New York / 10011
323.793.3369

**Comment by sonylnagale** [comment]
Hey Ella, take a look at what I just committed. That's how to get the value of the rating.

**Comment by sonylnagale** [comment]
Hey Ella,

This goes totally against protocol, but you're a special case, so whenever you're ready to chat you can text me at 323.793.3369.
